QueryBuilder.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace CanalTP\NavitiaPhp;
+
+class QueryBuilder
+{
+    /**
+     * @var string
+     */
+    private $coverage = null;
+
+    /**
+     * @var string
+     */
+    private $api = null;
+
+    /**
+     * @param string $coverage
+     *
+     * @return self
+     */
+    public function coverage($coverage)
+    {
+        $this->coverage = $coverage;
+
+        return $this;
+    }
+
+    /**
+     * @param string $api
+     *
+     * @return self
+     */
+    public function api($api)
```

Comment:
I don't really like this because you have a "put everything" method, and I don't see what your application bring to user.
Plus, it's more ->path() than ->api()